### PR TITLE
Card Component

### PR DESCRIPTION
**Pull request by horsemaker**
Added card component to the component library.

### CODE REVIEW

**Review by srishti-maurya — APPROVED** [review]

**Review by srishti-maurya — COMMENTED** [review]
UI is innovative

**Review by omgaikwad — COMMENTED** [review]
UI looks unique, cards look elegant, code is good!

**Comment by t007rushi** [comment]
Add comments in HTML else Everything looks Clean as always.Merge it.

**Comment by preethasri** [comment]
Looks neat and clean yash 👍

File: components/cards/card.css [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+.card-vertical {
+	max-width: 350px;
+	display: flex;
+	flex-direction: column;
+	gap: 0.75rem;
+}
+
+.card-horizontal {
+	max-width: 400px;
```
Comment (mimansha-swarup) [inline components/cards/card.css:19]: how about using rem or em instead of px

**Review by mimansha-swarup — COMMENTED** [review]
looks good 👍

**Review by vijaytembugade — APPROVED** [review]
Awesome cards. 
Good to go

File: components/cards/card.css (hunk 1 above)
Comment (horsemaker) [reply components/cards/card.css:19]: oh yeah, thanks for pointing it out! 👍

**Pull request merged**